spec/integration/date_time_properties_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+describe ActiveFedora::Base do
+  before do
+    class Foo < ActiveFedora::Base
+      property :date, predicate: ::RDF::DC.date do |index|
+        index.type :date
+      end
+      property :single_date, multiple:false, predicate: ::RDF::URI.new('http://www.example.com/single_date') do |index|
+        index.type :date
+      end
+    end
+  end
+
+  after do
+    Object.send(:remove_const, :Foo)
+  end
+
+  let(:date) { DateTime.parse("2015-10-22T10:20:03.653+01:00") }
+  let(:date2) { DateTime.parse("2015-10-22T15:34:20.323-11:00") }
+  subject { Foo.create(date: [date], single_date: date2) }
+
+  describe "saving and loading in Fedora" do
```

Comment:
It seems to reset attributes after save (or create). At least the time zone gets reset with create alone and that test fails. I could add a .reload in there though just in case.